modules/oc-mirror-about.adoc

```diff
@@ -24,3 +24,10 @@ The first time you run the oc-mirror plugin, it populates your mirror registry w
 ====
 When using the oc-mirror CLI plugin to populate a mirror registry, any further updates to the mirror registry must be made using the oc-mirror tool.
 ====
+
+[NOTE]
+====
+Operator Lifecycle Manager (OLM) manages the dependency resolution and upgrade lifecycle of running Operators. In many ways, the problems OLM faces are similar to other system or language package managers, such as yum and rpm.
```

Comment:
Operators being mirrored might have one or more [dependencies](https://olm.operatorframework.io/docs/concepts/olm-architecture/dependency-resolution/#m-docsconceptsolm-architectureoperator-catalogcreating-an-update-graph).
oc-mirror does a best effort to discover the dependencies, but it is highly recommended to include the dependencies in the image set configuration file explicitly.

Reply:
@subhtk - Could you move this out of about section ?